### ISSUE TITLE
Отображение информации об авторе и времени публикации

### ISSUE THREAD

**Issue by Zelebober** [umputun/remark#112]
В мобильной версии, особенно у комментариев не первого уровня, ломается дизайн заголовка с именем автора и временем публикации. Лучше будет разделить верстку этого блока на две строчки.

**Пример**

Сейчас:
![default](https://user-images.githubusercontent.com/4181413/41923229-e27f84be-79a1-11e8-987d-4f6b684ad3ae.png)

С разделением на строки:
![default](https://user-images.githubusercontent.com/4181413/41923231-e2c72864-79a1-11e8-8153-4887be4b5259.png)

**Comment by umputun** [comment]
Pls, use English for your issues and comments. Not all of the contributors can read Russian.

**Comment by slawiko** [comment]
@Zelebober don't hesitate to provide your styles. I see you did needed fix in attached pictures. Please share, because my proposal is to add `display: inline-block` for username and time elements. It will not fix issue in some extreme cases (too small window, or Nokia 3310) but do we need to support such cases?

**Comment by umputun** [comment]
@slawiko we don't care about Nokia 3310 ;)

**Comment by slawiko** [comment]
I do not think that it's a real issue. There is no problem if comment looks like issuer reported. But for cleaning my conscience I tried add `display: inline-block` for both timestamp and username.
Here are results:

Before 
![before](https://user-images.githubusercontent.com/10272192/42243936-a2de0ce8-7f1b-11e8-8383-78d29d8ff574.png)
After 
![after](https://user-images.githubusercontent.com/10272192/42243950-b0f30b08-7f1b-11e8-9d23-801e93458704.png)

Does anyone think that it's more nice? I do not. 

I did not change the layout, because of meaningless: someone will open app in smaller window and will see "broken" styling again. So my suggestion is to close this

But while working I've made refactoring a bit (#96) and I'll create PR

**Comment by Zelebober** [comment]
@slawiko for such a small screen resolution, I suggest using `@media`, so that the username and timestamp are placed in one line. It will look good.
